projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/JobImageBuilder.java

```diff
@@ -89,6 +89,9 @@ public class JobImageBuilder {
   @Value("${datajobs.deployment.builder.serviceAccountName}")
   private String builderServiceAccountName;
 
+  @Value("${dataJob.readOnlyRootFileSystem:false}")
+  private boolean readOnlyRootFilesystem;
```

Comment:
not needed anymore.

Reply:
dropped it, I assumed the Google Java format github action catches unused variables but I guess not.